cypress/integration/dashboard/dashboard_filtering_spec.js

```diff
@@ -0,0 +1,315 @@
+/**
+ * dashboard_filtering test suite description:
+ * 1) Create a new dashboard, populate with visualizations
+ * 2) Set a filter that excludes all data, and check the visualizations for proper updates
+ * 3) Set the existing filter as pinned, re-check the visualizations
+ * 4) Remove the filter, and check the visualizations for proper updates
+ * 5) Create a new dashboard, and populate with a pie graph
+ * 6) Perform apply different filters on the pie graph and check the visualization for proper updates
```

Comment:
No blocker. 1) you use 'the visualizations' in line 4 and 'the visualization' in line 6. Need to be consistent.  2) Perform and apply are both verbs. Should we delete either one of them? I think you probably need to polish your words a bit.

Reply:
Thank you for catching these writing errors! For the different usage of `the visualizations` and `the visualization`, line 6 is only referring to checking if the single pie graph added in line 5 had properly updated rather than the group of visualizations being tested in lines 1-4. To eliminate any ambiguity, I replaced the `the visualization` in line 6 with `the pie graph`. I also cut the `Perform` in line 6 to have the sentence make grammatical sense.

I also did a proof reading of the text in my added files, so I hopefully caught most of the other writing errors!